packages/devtools_app/lib/src/screens/performance/panes/queued_microtasks/queued_microtasks_view.dart

```diff
@@ -0,0 +1,278 @@
+            padding: const EdgeInsets.only(left: defaultSpacing),
+            alignment: Alignment.centerLeft,
+            child: const Row(
+              children: [
+                Text('Stack trace captured when microtask was enqueued'),
+              ],
+            ),
+          ),
+        ),
+        Row(
+          children: [
+            Expanded(
+              child: Padding(
+                padding: const EdgeInsets.symmetric(
+                  vertical: denseRowSpacing,
+                  horizontal: defaultSpacing,
+                ),
+                child: SelectableText(
+                  style: theme.fixedFontStyle,
+                  _selectedMicrotask!.stackTrace.toString(),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class QueuedMicrotasksTabView extends StatefulWidget {
+  const QueuedMicrotasksTabView({super.key, required this.controller});
+
+  final QueuedMicrotasksController controller;
+
+  @override
+  State<QueuedMicrotasksTabView> createState() =>
+      _QueuedMicrotasksTabViewState();
+}
+
+class _QueuedMicrotasksTabViewState extends State<QueuedMicrotasksTabView>
+    with AutoDisposeMixin {
+  static final _dateTimeFormat = DateFormat('HH:mm:ss.SSS (MM/dd/yy)');
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: widget.controller.status,
+      builder: (context, status, _) {
+        if (status == QueuedMicrotasksControllerStatus.empty) {
+          return const RefreshQueuedMicrotasksInstructions();
+        } else if (status == QueuedMicrotasksControllerStatus.refreshing) {
+          return Center(
+            child: Text(
+              style: Theme.of(context).regularTextStyle,
+              'Refreshing...',
+            ),
+          );
+        } else {
+          return ValueListenableBuilder(
+            valueListenable: widget.controller.queuedMicrotasks,
+            builder: (_, queuedMicrotasks, _) {
+              assert(queuedMicrotasks != null);
+
+              final indexedMicrotasks = queuedMicrotasks!.microtasks!.indexed
+                  .cast<IndexedMicrotask>()
+                  .toList();
+              final formattedTimestamp = _dateTimeFormat.format(
+                DateTime.fromMicrosecondsSinceEpoch(
+                  queuedMicrotasks.timestamp!,
+                ),
+              );
+              return Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.symmetric(
+                      vertical: denseRowSpacing,
+                      horizontal: defaultSpacing,
+                    ),
+                    child: Text(
+                      'Viewing snapshot that was taken at $formattedTimestamp.',
+                    ),
+                  ),
+                  Expanded(
+                    child: SplitPane(
```

Comment:
Look at other uses of SplitPane for an example, but usually we include borders using the OutlineDecoration widget around each of the split pane children

Reply:
Done.